### PR TITLE
Sample-list

### PR DESCRIPTION
**Pull request by sselin-co**
To test this, first make a handful of new samples in the admin dashboard.

### CODE REVIEW

**Comment by sselin-co** [comment]
I attempted some new testing assertations to look at the template context variable and confirm the right template is rendered, but  I haven't had much luck actually getting them to work. I removed them for now till I can figure out the syntax

**Review by shepkeira — CHANGES_REQUESTED** [review]
You need to do a database backup, to maintain the database changes/migrations

**Comment by shepkeira** [comment]
PS do you mind if we merge Sayo's database changes? It shouldn't affect yours you'll just need to merge dev into your branch, and then redo the migrate and backup so both database changes are in affect

**Comment by sselin-co** [comment]
I'll do the database backup in a few hours, and yes I can certainly wait till Sayo's changes have been merged

File: LIMS_IMAGE/web/src/settings.py [hunk 1]
```diff
@@ -120,7 +120,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'PST8PDT'
```
Comment (shepkeira) [inline LIMS_IMAGE/web/src/settings.py:123]: maybe add a comment that this is PST with daylight saving accounted for?

Comment (sselin-co) [reply LIMS_IMAGE/web/src/settings.py:123]: Good catch, I'll add that

**Comment by shepkeira** [comment]
Sayo's is merged, you are good to merge development into your branch

**Review by Rock-it-science — APPROVED** [review]
Looks great!

**Review by shepkeira — APPROVED** [review]
Amazing 🎉

**Pull request merged**